core/src/main/java/org/akaza/openclinica/service/OdmImportServiceImpl.java

```diff
@@ -88,10 +102,9 @@
     protected final Logger logger = LoggerFactory.getLogger(getClass().getName());
 
     public OdmImportServiceImpl(DataSource dataSource) {
-        super();
-        this.dataSource = dataSource;
     }
 
+    @Transactional
     public void importOdmToOC(org.cdisc.ns.odm.v130_sb.ODM odm) {
```

Comment:
Can you just do ODM here instead of org.cdisc.ns.odm.v130_sb.ODM?
Makes the code less readable.